src/components/SearchComponent.tsx

```diff
@@ -52,6 +54,8 @@ const SearchComponent: React.FC<SearchComponentProps> = ({
     }
   }, [])
 
+  const openInNewTab = location === "/editor" || location === "/ai"
```

Comment:
don't use verbs for variable names, verbs are for function names. `shouldOpenInNewTab` is a var name

```suggestion
  const shouldOpenInNewTab = location === "/editor" || location === "/ai"
```